docs/source/conf.py

```diff
@@ -69,7 +69,7 @@
 
 intersphinx_mapping = {
     "kedro-viz": ("https://docs.kedro.org/projects/kedro-viz/en/v6.6.1/", None),
-    "kedro-datasets": ("https://docs.kedro.org/projects/kedro-datasets/en/kedro-datasets-2.0.0/", None),
+    "kedro-datasets": ("https://docs.kedro.org/projects/kedro-datasets/en/kedro-datasets-2.0.0.post1/", None),
```

Comment:
```suggestion
    "kedro-datasets": ("https://docs.kedro.org/projects/kedro-datasets/en/kedro-datasets-4.0.0/", None),
```
Actually, why is this so old, anyway.